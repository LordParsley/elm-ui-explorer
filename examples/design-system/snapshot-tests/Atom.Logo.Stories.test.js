
    const tree = `<div>
    <div class="snapshot" id="Default">
        <img alt="Bulma: a modern CSS framework based on Flexbox" height="28" src="http://bulma.io/images/bulma-logo.png" width="112">
    </div>
</div>`;
    test("Atom.Logo.Stories", () => {
      expect(tree).toMatchSnapshot();
    });
    