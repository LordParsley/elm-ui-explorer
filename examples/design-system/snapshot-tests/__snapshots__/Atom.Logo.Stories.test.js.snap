// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom.Logo.Stories 1`] = `
"<div>
    <div class=\\"snapshot\\" id=\\"Default\\">
        <img alt=\\"Bulma: a modern CSS framework based on Flexbox\\" height=\\"28\\" src=\\"http://bulma.io/images/bulma-logo.png\\" width=\\"112\\">
    </div>
</div>"
`;
